<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-C7858AT" failures="2" tests="2" name="Test(Failed)" time="0.144" errors="0" timestamp="2023-09-07T17:39:52 IST">
  <testcase classname="Failed_Xml_Suit.Test3" name="Test_Case6" time="0.026">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Failed_Xml_Suit.Test3.Test_Case6(Test3.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Test_Case6 -->
  <testcase classname="Failed_Xml_Suit.Test2" name="Test_Case3" time="0.004">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Failed_Xml_Suit.Test2.Test_Case3(Test2.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Test_Case3 -->
  <testcase classname="Failed_Xml_Suit.Test3" name="Test_Case5" time="0.0">
    <ignored/>
  </testcase> <!-- Test_Case5 -->
  <testcase classname="Failed_Xml_Suit.Test2" name="Test_Case4" time="0.0">
    <ignored/>
  </testcase> <!-- Test_Case4 -->
</testsuite> <!-- Test(Failed) -->
